Boost 1.67.0 failure: error: called object type 'nullptr_t' is not a function or function pointer

```
In file included from /usr/local/include/boost/preprocessor/iteration/detail/iter/forward1.hpp:67:
In file included from /usr/local/include/boost/function/detail/function_iterate.hpp:14:
In file included from /usr/local/include/boost/function/detail/maybe_include.hpp:43:
/usr/local/include/boost/function/function_template.hpp:159:33: error: called object type 'nullptr_t' is not a function or function pointer
          BOOST_FUNCTION_RETURN((*f)(BOOST_FUNCTION_ARGS));
                                ^~~~
/usr/local/include/boost/function/function_template.hpp:81:36: note: expanded from macro 'BOOST_FUNCTION_RETURN'
#  define BOOST_FUNCTION_RETURN(X) X
                                   ^
/usr/local/include/boost/function/function_template.hpp:934:53: note: in instantiation of member function 'boost::detail::function::void_function_obj_invoker4<nullptr_t, void, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &>::invoke' requested here
        { { &manager_type::manage }, &invoker_type::invoke };
                                                    ^
/usr/local/include/boost/function/function_template.hpp:725:13: note: in instantiation of function template specialization 'boost::function4<void, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &>::assign_to<nullptr_t>' requested here
      this->assign_to(f);
            ^
/usr/local/include/boost/function/function_template.hpp:1070:5: note: in instantiation of function template specialization 'boost::function4<void, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &>::function4<nullptr_t>' requested here
    base_type(f)
    ^
/wrkdirs/usr/ports/graphics/pcl-pointclouds/work/pcl-pcl-1.8.1/registration/include/pcl/registration/registration.h:126:11: note: in instantiation of function template specialization 'boost::function<void (const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &, const pcl::PointCloud<pcl::PointNormal> &, const std::__1::vector<int, std::__1::allocator<int> > &)>::function<nullptr_t>' requested here
        , update_visualizer_ (NULL)
          ^
/wrkdirs/usr/ports/graphics/pcl-pointclouds/work/pcl-pcl-1.8.1/registration/include/pcl/registration/icp.h:137:7: note: in instantiation of member function 'pcl::Registration<pcl::PointNormal, pcl::PointNormal, float>::Registration' requested here
      IterativeClosestPoint () 
      ^
/wrkdirs/usr/ports/graphics/pcl-pointclouds/work/pcl-pcl-1.8.1/recognition/src/face_detection/rf_face_detector_trainer.cpp:479:68: note: in instantiation of member function 'pcl::IterativeClosestPoint<pcl::PointNormal, pcl::PointNormal, float>::IterativeClosestPoint' requested here
    pcl::IterativeClosestPoint<pcl::PointNormal, pcl::PointNormal> reg;
                                                                   ^
```

Build log: http://beefy11.nyi.freebsd.org/data/head-i386-default/p472945_s335462/logs/pcl-pointclouds-1.8.1_5.log (IPv6-only)

--- a/registration/include/pcl/registration/registration.h
+++ b/registration/include/pcl/registration/registration.h
@@ -124,7 +124,6 @@ namespace pcl
         , source_cloud_updated_ (true)
         , force_no_recompute_ (false)
         , force_no_recompute_reciprocal_ (false)
-        , update_visualizer_ (NULL)
         , point_representation_ ()
       {
       }

